README.md

```diff
@@ -1 +1,37 @@
-# Portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-world
+
+> My first project as a Microverse_student.
+
+
+## Built With
+
+- Html
+- Css
+- Linters
+
+
+## Author
+
+👤 **Benjamin Ejike**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
Great job @osbentech 🤝.

- [ ] You may please customize these handles with your own initials.